### PR TITLE
Update EmailValidationBehavior.shared.cs

### PR DESCRIPTION
**Pull request by Rashidovicc**
Fixed exception message for syntax. Also, the case should be standard case and the old version should be pascal case, and such minor errors should be corrected.

 ### Description of Change ###

 

 ### Linked Issues ###
 

 - Fixes #

 ### PR Checklist ###
 
 - [ ] Has a linked Issue, and the Issue has been `approved`(bug) or `Championed` (feature/proposal)
 - [ ] Has tests (if omitted, state reason in description)
 - [ ] Has samples (if omitted, state reason in description)
 - [ ] Rebased on top of `main` at time of PR
 - [x] Changes adhere to [coding standard](https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#contributing-code---best-practices)
 - [ ] Documentation created or updated: https://github.com/MicrosoftDocs/CommunityToolkit/pull/XYZ 

 ### Additional information ###

### CODE REVIEW

File: src/CommunityToolkit.Maui/Behaviors/Validators/EmailValidationBehavior.shared.cs [hunk 1]
```diff
@@ -111,10 +111,10 @@ static string domainMapper(Match match)
 
 			if (domainName.StartsWith('-'))
 			{
-				throw new ArgumentException("Domain Name Cannot Start With Hyphen");
+				throw new ArgumentException("Domain name cannot start with hyphen");
```
Comment (VladislavAntonyuk) [inline src/CommunityToolkit.Maui/Behaviors/Validators/EmailValidationBehavior.shared.cs:114]:
```suggestion
				throw new ArgumentException("Domain name cannot start with hyphen.");
```

**Review by VladislavAntonyuk — APPROVED** [review]

**Review by VladislavAntonyuk — APPROVED** [review]

**Comment by VladislavAntonyuk** [comment]
Please follow readme before creating the PR

**Comment by Rashidovicc** [comment]
Will this PR be merged?

**Comment by VladislavAntonyuk** [comment]
No, it is closed a not expected

**Comment by bijington** [comment]
@Rashidovicc We really do appreciate contributions but they need to be of value to the toolkit and the community. 

We are more than happy to explain the process of you really are keen on contributing. 

Can I ask what attracted you to contributing to our project?

**Comment by Rashidovicc** [comment]
What attracted me was the atmosphere, I asked myself a question, several
contributors were helping as much as they could, and I got into it. I also
want to join Hassan Habib's The-Standard-Uzbek repository and start
translating it. I apologize if I have spoken badly, but I want to
contribute to this project, even if it is a very small part.I hope you
understand me and I apologize again.

пн, 5 сент. 2022 г. в 19:20, Shaun Lawrence ***@***.***>:

**Pull request closed without merging**